### PR TITLE
3977 wildberries actions

### PR DESCRIPTION
**Pull request by alysonturing**
Actions
- List orders
- List orders stickers (regular and as pdf)
- Update order status

FYI: I couldn't test it properly because the API Key provided by the customer is not working anymore.

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4111"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/wildberries/actions/common/common.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+export default {
+  stickersRequestType: [
```
Comment (lcaresia) [inline components/wildberries/actions/common/common.ts:2]:
```suggestion
  STICKERS_REQEUEST_TYPE: [
```

File: components/wildberries/actions/common/common.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "code128",
+    "qr",
+  ],
+  orderStatus: [
```
Comment (lcaresia) [inline components/wildberries/actions/common/common.ts:6]:
```suggestion
  ORDER_STATUSES: [
```

File: components/wildberries/actions/list-order-stickers/list-order-stickers.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    orderIds: {
+      propDefinition: [
+        app,
+        "orderIds",
+      ],
+    },
+    type: {
+      label: "Type",
+      type: "string",
+      description: "Sticker type, default: code128.",
```
Comment (lcaresia) [inline components/wildberries/actions/list-order-stickers/list-order-stickers.ts:22]:
```suggestion
      description: "Sticker type, default: `code128`.",
```

File: components/wildberries/actions/list-order-stickers/list-order-stickers.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import app from "../../app/wildberries.app";
+import common from "../common/common";
+import { defineAction } from "@pipedream/types";
+
+export default defineAction({
+  name: "List Order Stickers",
+  description: "List order stickers. [See docs](https://suppliers-api.wildberries.ru/swagger/index.html#/Marketplace/post_api_v2_orders_stickers)",
```
Comment (lcaresia) [inline components/wildberries/actions/list-order-stickers/list-order-stickers.ts:7]:
```suggestion
  description: "List order stickers. [See docs here](https://suppliers-api.wildberries.ru/swagger/index.html#/Marketplace/post_api_v2_orders_stickers)",
```

File: components/wildberries/actions/list-order-stickers/list-order-stickers.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      label: "Type",
+      type: "string",
+      description: "Sticker type, default: code128.",
+      default: "code128",
+      options: common.stickersRequestType,
+    },
+    asPdf: {
+      type: "boolean",
+      label: "List as PDF",
+      description: "Set true for use the PDF API [See docs](https://suppliers-api.wildberries.ru/swagger/index.html#/Marketplace/post_api_v2_orders_stickers_pdf).",
```
Comment (lcaresia) [inline components/wildberries/actions/list-order-stickers/list-order-stickers.ts:29]:
```suggestion
      description: "Set true for use the PDF API [See docs here](https://suppliers-api.wildberries.ru/swagger/index.html#/Marketplace/post_api_v2_orders_stickers_pdf).",
```

File: components/wildberries/actions/list-orders/list-orders.ts [hunk 6]
```diff
@@ -0,0 +1,67 @@
+import app from "../../app/wildberries.app";
+import { defineAction } from "@pipedream/types";
+
+export default defineAction({
+  name: "List Orders",
+  description: "Returns a list of orders. [See docs](https://suppliers-api.wildberries.ru/swagger/index.html#/Marketplace/get_api_v2_orders)",
```
Comment (lcaresia) [inline components/wildberries/actions/list-orders/list-orders.ts:6]:
```suggestion
  description: "Returns a list of orders. [See docs here](https://suppliers-api.wildberries.ru/swagger/index.html#/Marketplace/get_api_v2_orders)",
```

File: components/wildberries/actions/update-order-status/update-order-status.ts [hunk 7]
```diff
@@ -0,0 +1,46 @@
+import app from "../../app/wildberries.app";
+import { defineAction } from "@pipedream/types";
+
+export default defineAction({
+  name: "Update Order Status",
+  description: "Update a order status. [See docs](https://suppliers-api.wildberries.ru/swagger/index.html#/Marketplace/put_api_v2_orders)",
```
Comment (lcaresia) [inline components/wildberries/actions/update-order-status/update-order-status.ts:6]:
```suggestion
  description: "Update a order status. [See docs here](https://suppliers-api.wildberries.ru/swagger/index.html#/Marketplace/put_api_v2_orders)",
```

File: components/wildberries/actions/update-order-status/update-order-status.ts [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+    status: {
+      propDefinition: [
+        app,
+        "status",
+      ],
+    },
+    sgtin: {
+      type: "any",
+      label: "SGTIN",
+      description: "Array required only for pharmaceutical products when they are transferred to status `Customer received the goods`.\n\n**Example:** `[{ code: string, numerator: integer, denominator: integer, sid: integer }]`\n\n[See docs](https://suppliers-api.wildberries.ru/swagger/index.html#/Marketplace/put_api_v2_orders)",
```
Comment (lcaresia) [inline components/wildberries/actions/update-order-status/update-order-status.ts:27]:
```suggestion
      description: "Array required only for pharmaceutical products when they are transferred to status `Customer received the goods`.\n\n**Example:** `[{ code: string, numerator: integer, denominator: integer, sid: integer }]`\n\n[See docs here](https://suppliers-api.wildberries.ru/swagger/index.html#/Marketplace/put_api_v2_orders)",
```

**Review by lcaresia — CHANGES_REQUESTED** [review]
Thanks for your contribution.

Just some fixes!

**Review by lcaresia — APPROVED** [review]
LGTM!

**Pull request merged**